src/System.IdentityModel.Tokens.Jwt/JwtPayload.cs

```diff
@@ -299,6 +299,18 @@ public DateTime ValidTo
             }
         }
 
+		/// <summary>
```

Comment:
Hi @macaba, thanks for the contribution.
The formatting is slightly off here and at the same spot in JwtSecurityToken class.

Reply:
Thanks, resolved.